### DISCUSSION TITLE
New CI service

### DISCUSSION THREAD

**Discussion by tomdottom** [Wanderu/pytest-bandit#25] [General]
As mentioned in #23 travis-ci.org is shutting down and making changes to how they support OSS projects.

- Is there current offering for OSS projects enough for `pytest-bandit` (would it even qualify)?; or
- Should an alternative CI service be considered?

**Comment by mattwwarren** [comment]
Pulling my comment from the issue here:

Seems like we could try to run with travis-ci.com or set up Actions. I'm tempted to experiment with the latter, which seems to have plenty of boilerplate, but open to influence.
